src/main/java/com/synopsys/integration/alert/channel/ChannelFreemarkerTemplatingService.java

```diff
@@ -30,39 +30,40 @@
 import org.apache.commons.lang3.StringUtils;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
 
 import com.synopsys.integration.alert.common.AlertConstants;
-import com.synopsys.integration.alert.common.enumeration.EmailPropertyKeys;
 
 import freemarker.template.Configuration;
 import freemarker.template.Template;
 import freemarker.template.TemplateException;
 import freemarker.template.TemplateExceptionHandler;
 
+@Component
 public class ChannelFreemarkerTemplatingService {
     private final Logger logger = LoggerFactory.getLogger(ChannelFreemarkerTemplatingService.class);
-    private final String templateDirectory;
-    private final Configuration configuration;
 
-    public ChannelFreemarkerTemplatingService(final String templateDirectory) throws IOException {
-        this.templateDirectory = templateDirectory;
-        this.configuration = createFreemarkerConfig();
+    public String getTemplatePath(final String templatesDirectory, final String channelDirectory) {
+        if (StringUtils.isNotBlank(templatesDirectory)) {
+            return String.format("%s/%s", templatesDirectory, channelDirectory);
+        }
+        return String.format("%s/src/main/resources/%s/templates", System.getProperties().getProperty("user.dir"), channelDirectory);
```

Comment:
For production the templates are exported to a different location to be accessible in the docker container. See _distributions_ in the build.gradle.

Reply:
We should refactor this to use AlertProperties.getAlertTemplatesDir() at this point.  This is already used to get the templates. We should also refactor the build.gradle to pass the property in via the command line.  Also update the application properties file used for the tests to set this alert.templates.dir as well.  This way we can remove line 50.

Reply:
When you say update build.gradle, will I only be updating the runServer command to add this property value? I'm not aware of anywhere else I'd need to make changes for this.